spring-websocket/src/main/java/org/springframework/web/socket/messaging/StompSubProtocolHandler.java

```diff
@@ -399,8 +399,6 @@ private void sendErrorMessage(WebSocketSession session, Throwable error) {
 		headerAccessor.setMessage(error.getMessage());
 
 		byte[] bytes = this.stompEncoder.encode(headerAccessor.getMessageHeaders(), EMPTY_PAYLOAD);
```

Comment:
primitive data type is not implementing interface `autocloseable`, seems impossible to use `try-with-resources`, right?.

not possible in `L:413` ether, so idk if we still need this comment, as misleading.

<img width="821" alt="image" src="https://github.com/user-attachments/assets/3dc7787b-012d-4396-809a-6a2c6baf7539" />